arch/arm/src/cxd56xx/cxd56_dmac.c

```diff
@@ -328,22 +332,25 @@ static struct dmac_register_map *get_device(int ch)
     case 2: return (struct dmac_register_map *)DMAC2_REG_BASE;
     case 3: return (struct dmac_register_map *)DMAC3_REG_BASE;
     }
+
     return NULL;
 }
 
 static struct dmac_ch_register_map *get_channel(int ch)
 {
   struct dmac_register_map *dev = get_device(ch);
   if (dev == NULL)
-     return NULL;
+    {
+      return NULL;
+    }
 
   if (is_dmac(2, dev))
     {
-        return &dev->channel[ch - 7];
+      return &dev->channel[ch - 7];
```

Comment:
can we separate style changes into a separate commit?

Reply:
Done, the style change separate to a new patch